app/controllers/application_controller/ci_processing.rb

```diff
@@ -143,7 +143,7 @@ def process_elements(elements, klass, task, display_name = nil, order_field = ni
   end
 
   def explorer_controller?
-    %w[vm_cloud vm_infra vm_or_template infra_networking].include?(controller_name)
+    %w[vm_cloud vm_infra vm_or_template infra_networking automation_manager].include?(controller_name)
```

Comment:
shouldn't be `configuration_manager`?

Reply:
no, `configuration_manger` is no longer an explorer controller but shares some code with `automation_manager` controller and that is explorer style screen, so i need to add this here so i can redirect appropriately form shared methods.

Reply:
OH, right